ui/apps/everest/src/pages/database-form/database-form-body/steps/first/first-step.tsx

```diff
@@ -220,20 +217,24 @@ export const FirstStep = ({ loadingDefaultsForEdition }: StepProps) => {
     setDbEngineDataForEngineType();
   }, [setDbEngineDataForEngineType]);
 
+  const { permittedNamespaces, isFetching } = useGetPermittedNamespaces({
```

Comment:
`allowedNamespaces` was a now deprecated features for monitoring endpoints and storage locations. Permitted namespaces show the namespaces that have create rights for a certain resource. I changed the name so that it's not confusing